src/main/java/org/team1540/rooster/wrappers/ChickenVictor.java

```diff
@@ -16,7 +16,7 @@
 import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
 import com.ctre.phoenix.motorcontrol.can.VictorSPX;
 
-public class ChickenVictor extends VictorSPX implements ChickenController {
```

Comment:
...no. We need to update the ChickenController interface, not the other way around.